photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataSetPartitioner.scala

```diff
@@ -93,11 +93,11 @@ object RandomEffectDataSetPartitioner {
   /**
    * Generate a partitioner for one random effect model.
    *
-   * A random effect model is composed of multiple sub-models, each trained on data points for a single item.
-   * We collect the training vector ids that correspond to the random effect, then build an id to partition map.
-   * Data should be distributed across partitions as equally as possible. Since some items have more data points
-   * than others, this partitioner uses simple 'bin packing' for distributing data load across partitions (using
-   * minHeap).
+   * A random effect type (e.g. "per-user") is instantiated for many items (e.g. "user"), and of these instantiations
+   * is trained with training vectors for that item. We collect the training vector ids that correspond to a random
+   * effect type, then build an id to partition map. Data should be distributed across partitions as equally as
+   * possible. Since some items have more data points than others, this partitioner uses simple 'bin packing' for
+   * distributing data load across partitions (using minHeap).
```

Comment:
There's a typo here ("... and of these instantiations ..."). I think the previous sentence was clearer than the new one.